src/core/common/utils/getHost.ts

```diff
@@ -0,0 +1,4 @@
+export default function getHost(url: string) {
+  const split = url.split("/");
+  return split[2];
+}
```

Comment:
can we do something like the following here?

```
const path = new URL(url);
const host = path.hostname;
```

Reply:
Hmm I guess, though this implementation can be used server side as well, so as long as the test passes, I think it's fine?

Reply:
You can use `URL` class server side as well.

I ran the following in node server side:
```
const rawURL = "https://www.theverge.com/2022/2/2/22914970/facebook-app-loses-daily-users-first-time-earnings";

const url = new URL(rawURL);
console.log("url based hostname:", url.hostname);

const split = rawURL.split("/");
console.log("split based hostname:", split[2]);
```

Got this result:
```
url based hostname: www.theverge.com
split based hostname: www.theverge.com
```

The only reason I suggest it is that URL will validate the link with all edge cases. For instance, web browsers will accept `http:/` as well as `http://`. The `new URL(...)` library does this as well, but the split based method will fail and return 2022 for the above example `rawURL`.

Also, `URL` will throw other errors if the URL is malformed in other weird ways and give an informative message.

Reply:
Ah didn't know it works serverside. I guess I can use  `new URL(url).host` directly then, thanks!

Reply:
Actually no, it doesn't support omitting the protocol `//domain` which browsers do accept and is standardized. `coral-party` repository e.g. is depending on it.

Reply:
Browsers only accept `//domain.com` to point to `file://domain.com`?

Reply:
Anyways, this is fine for now. Not holding a review over a two line change.